### PR TITLE
Feature/555 change header title by using props

### PR DESCRIPTION
**Pull request by dkrakau**
### Description

ISSUE: #555 change header title by using props

## Checklist

- [x] My code follows the style guidelines
- [ ] I have documented my code
- [x] My changes generate no new warnings
- [x] I have linked the Issue
- [ ] The Pull Request covers all the Checkpoints from the tagged Issue
- [x] I have noted the hours of work i have put in the issue

## I have edited following Parts of the project

- [x] Frontend
- [ ] Backend
- [ ] Github
- [ ] Root

## If tests are required did you make them?

- [ ] No tests required
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by AconexOfficial — APPROVED** [review]

File: frontend/src/components/ResultRoutes/ResultRoutes.tsx [hunk 1]
```diff
@@ -168,8 +168,8 @@ const ResultRoutes: React.FC<ResultRoutesProps> = ({ origin, destination }) => {
   }, [activeSlideIndex]);
 
   return (
-    <>
-      <Header />
+    <IonPage id="main-content">
```
Comment (timvahlbrock) [inline frontend/src/components/ResultRoutes/ResultRoutes.tsx:171]: Why did you remove this? Removing the id is okay, but I think IonPage Directive is needed for proper navigation.

**Review by timvahlbrock — CHANGES_REQUESTED** [review]

**Review by timvahlbrock — APPROVED** [review]
